components/raftstore/src/store/fsm/store.rs

```diff
@@ -154,6 +162,34 @@ pub struct StoreMeta {
     pub damaged_ranges: HashMap<String, (Vec<u8>, Vec<u8>)>,
 }
 
+impl StoreRegionMeta for StoreMeta {
```

Comment:
why change v1's code?

Reply:
So that structs can accepts StoreMeta from both v1 and v2.